migen/build/platforms/sinara/kasli_soc.py

```diff
@@ -0,0 +1,314 @@
+from migen.build.generic_platform import *
+from migen.build.xilinx import XilinxPlatform
+
+
+_io = [
+    ("clk500", 0, Pins("B24"), IOStandard("LVCMOS25")),
+
+    ("user_led", 0, Pins("AF19"), IOStandard("LVCMOS25")),
+    ("user_led", 1, Pins("AF23"), IOStandard("LVCMOS25")),
+    ("user_led", 2, Pins("A23"), IOStandard("LVCMOS25")),
+    ("error_led", 0, Pins("D20"), IOStandard("LVCMOS25")),
```

Comment:
Some of these LEDs are PS.

Reply:
Right, `LED_USER2` and `LED_ERR` are PS.